src/UIComponents/DataDisplay.jsx

```diff
@@ -86,122 +85,110 @@ class DataDisplay extends Component {
-                            {data.length > 0 &&
-                              Object.keys(row).map(key => {
-                                return (
-                                  <td
-                                    key={key}
-                                    style={this.getColumnCellStyle(key)}
-                                    onClick={() => setCurrentColumn(key)}
-                                  >
-                                    {row[key]}
-                                  </td>
-                                );
-                              })}
-                          </tr>
-                        );
-                      })}
-                  </tbody>
-                </table>
+                    </tbody>
+                  </table>
+                </div>
               </div>
-            </div>
-          )}
-          {!this.state.showRawData && (
-            <button type="button" onClick={this.toggleRawData}>
-              show data
-            </button>
-          )}
-          <div style={styles.mediumText}>
-            There are {this.props.data.length} rows of data.
+            )}
           </div>
+        </div>
+        {!this.state.showRawData && (
+          <button type="button" onClick={this.toggleRawData}>
+            show data
+          </button>
+        )}
+        <div style={styles.mediumText}>
+          There are {this.props.data.length} rows of data.
+        </div>
 
-          <div style={styles.mediumText}>
-            There are {this.props.emptyCellDetails.length} empty cells.
-          </div>
-          {this.state.showEmptyCellDetails && (
-            <div>
-              <button type="button" onClick={this.toggleEmptyCellDetails}>
-                hide empty cell details
-              </button>
-              {this.props.emptyCellDetails.map((cellDetails, i) => {
-                return <p key={i}>{cellDetails}</p>;
-              })}
-            </div>
-          )}
-          {!this.state.showEmptyCellDetails &&
-            this.props.emptyCellDetails.length > 0 && (
-              <button type="button" onClick={this.toggleEmptyCellDetails}>
-                show empty cell details
-              </button>
-            )}
+        {/*<div style={styles.mediumText}>
+          There are {this.props.emptyCellDetails.length} empty cells.
         </div>
+        {this.state.showEmptyCellDetails && (
+          <div>
+            <button type="button" onClick={this.toggleEmptyCellDetails}>
+              hide empty cell details
+            </button>
+            {this.props.emptyCellDetails.map((cellDetails, i) => {
+              return <p key={i}>{cellDetails}</p>;
+            })}
+          </div>
+        )}
+        {!this.state.showEmptyCellDetails &&
+          this.props.emptyCellDetails.length > 0 && (
+            <button type="button" onClick={this.toggleEmptyCellDetails}>
+              show empty cell details
+            </button>
+          )*/}
```

Comment:
Do we want to keep this around as a comment?  May just rely on PR history if we need to revive it?